kernel/src/main/java/org/kframework/compile/RewriteAwareVisitor.java

```diff
@@ -69,6 +69,18 @@ public void apply(KApply k) {
                 isRHS = wasRHS;
                 isLHS = wasLHS;
                 apply(k.items().get(1));
+            } else if (k.klabel().name().equals("#fun3")) {
```

Comment:
Should this be `#fun3` or `#let`? I guess the next case was handling `#fun3` before, and now this case is handling `#fun3` and the next case is handling `#let`.

Reply:
Ah, actually, the cases are identical. Why don't we just do `.equals('#fun3') or .equals('#let')` and have one case?